js/views/message_view.js

```diff
@@ -10,7 +10,7 @@
 (function () {
   'use strict';
 
-  const { HTML } = window.Signal;
+  const { Signal } = window;
```

Comment:
Minor: I’ve started using `Signal.*` directly and whitelisted `Signal` as a `global` above to remind us which ones we use instead of using `window` catch-all.

Reply:
We're already in a great place when we eslintify a file, because anything that is global but doesn't have the `window` prefix is caught. Otherwise, it's easy to do a search in the file for `window.`.